_ml-commons-plugin/index.md

```diff
@@ -23,3 +23,12 @@ There are two reserved user roles that can use of the ML commons plugin.
 
 - `ml_full_access`: Full access to all ML features, including starting new ML tasks and reading or deleting models.
 - `ml_readonly_access`: Can only read ML tasks, trained models and statistics relevant to the model's cluster. Cannot start nor delete ML tasks or models.
+
+## ML node
+
+To prevent your cluster from failing when running ML tasks, you configure a dynamic node with the `ml` role. ML nodes will not store any shards and calculates resource requirements at runtime. To use an ML node, create a node in your `opensearch.yml` file. Change the name of the node to `ml-node` and define the node role as `ml`:
```

Comment:
`configure a dynamic node with the `ml` role.` , not sure if we should use "dynamic node" here. I think we should say the "node role" is dynamic. How about changing it to "configure a dedicated ML node with `ml` dynamic node role?"

Reply:
`ML nodes will not store any shards and calculates resource requirements at runtime.` This depends. We suggest user only configure `ml` node role for dedicated ML node, don't add `data` role so it won't store shards on it. But if user configure both `ml` and `data` roles for  node, it will store shards.

Reply:
`Change the name of the node to `ml-node` and define the node role as `ml`:` , actually `node.name` doesn't matter, user can use any name, just make sure no other nodes using the same name in cluster.